doc/src/manual/getting-started.md

```diff
@@ -14,16 +14,14 @@ io = IOBuffer()
 Base.banner(io)
 banner = String(take!(io))
 import Markdown
-Markdown.parse("```\n\$ julia\n\n$(banner)\njulia> 1 + 2\n3\n\njulia> ans\n3\n```")
+Markdown.parse("```\n\$ julia\n\n$(banner)\njulia> 1 + 2\n3\n```")
 ```
 
 To exit the interactive session, type `CTRL-D` (press the Control/`^` key together with the `d` key), or type
 `exit()`. When run in interactive mode, `julia` displays a banner and prompts the user for input.
 Once the user has entered a complete expression, such as `1 + 2`, and hits enter, the interactive
 session evaluates the expression and shows its value. If an expression is entered into an interactive
-session with a trailing semicolon, its value is not shown. The variable `ans` is bound to the
```

Comment:
I think it is ok to remove the use of `ans` in the docstrings but this seems like a quite innocent piece of documentation that can actually be pretty useful to know while getting started.  I would argue for putting this part back.

Reply:
I agree that it's a good and useful piece of documentation. It's the prominent position in the "Getting Started" section that seems a little out of place, and potentially distracting.

How about moving this explanation near the top of the "Variables" section, which is the next section after "Getting Started"? This seems like a natural place. A new user working their way through the docs will still see it just as early, so they can use `ans` as they play around on the REPL.